src/Tasks/Microsoft.Common.CurrentVersion.targets

```diff
@@ -4422,7 +4422,12 @@ Copyright (C) Microsoft Corporation. All rights reserved.
       <_ClickOnceRuntimeCopyLocalItems Include="@(NativeCopyLocalItems)"
                                       Condition="'%(NativeCopyLocalItems.CopyLocal)' == 'true'" />
       <_ClickOnceRuntimeCopyLocalItems Remove="@(_DeploymentReferencePaths)" />
-      <_ClickOnceFiles Include="@(ContentWithTargetPath);@(_DeploymentManifestIconFile);@(AppConfigWithTargetPath);@(NetCoreRuntimeJsonFilesForClickOnce);@(_ClickOnceRuntimeCopyLocalItems)"/>
+
+      <!-- Include items from None itemgroup for publishing -->
+      <_ClickOnceNoneItems Include="@(_NoneWithTargetPath->'%(FullPath)')" Condition="'%(_NoneWithTargetPath.CopyToOutputDirectory)'=='Always'"/>
```

Comment:
I think you can use an 'or' to combine these.

Also, should there be an extra condition on PreserveNewest, i.e., a timestamp check?

Reply:
Fixed the condition.

The CopyToOutputDirectory.PreserveNewest is metadata that is handled by the build targets to copy newer files only. From ClickOnce publishing perspective, once it's in the output folder, we will publish it.